tornado_stale_client/async_client.py

```diff
@@ -0,0 +1,87 @@
+# -*- coding: utf-8 -*-
+
+# This file is part of tornado-stale-client.
+# https://github.com/globocom/tornado-stale-client
+
+# Licensed under the MIT license:
+# http://www.opensource.org/licenses/MIT-license
+# Copyright (c) 2016, Globo.com <backstage@corp.globo.com>
+
+import logging
+
+from tornado import gen
+from tornado.httpclient import HTTPRequest
+
+from tornado_stale_client.base import BaseStaleHTTPClient
+
+
+class AsyncStaleHTTPClient(BaseStaleHTTPClient):
```

Comment:
Can we come up with a more expressive name? The current client is already asynchronous (if we consider http, not redis).

Calling this client "async" sounds like telling that the original client is not async.

Maybe we should fix the original renaming to "async" and come up with a name that makes it clear that the `redis` access is also async.

What do you think @rfloriano?